common-ui/src/main/res/layout/component_outline_text_input.xml

```diff
@@ -21,8 +21,7 @@
     android:orientation="vertical"
     android:paddingStart="@dimen/keyline_4"
     android:paddingTop="@dimen/keyline_5"
-    android:paddingEnd="@dimen/keyline_4"
-    android:paddingBottom="@dimen/keyline_5">
```

Comment:
This is strange; having this padding makes the bottom input view get squished. Flagged in another task: https://app.asana.com/0/1174433894299346/1203157856956586